<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataShowDtl</class>
 <widget class="QWidget" name="dataShowDtl">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#dataShowDtl{
background-color : rgb(244, 244, 244);
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>740</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>740</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>740</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: rgb(244, 244, 244);
}</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>481</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(42, 180, 255);
    font:25px; 
    font-family: 幼圆;
    font-weight:bold;
    color:white;
 
}
</string>
    </property>
    <property name="text">
     <string>数据报表</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>155</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>155</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit{
border: 1px solid gray;
border-radius: 10px;
font-family: Segoe UI;
font:15px;
background-color:white;
}
QDateEdit::up-button{
image: url(:/menu/res/upbutton.png);
}
QDateEdit::down-button{
image: url(:/menu/res/downbutton.png);
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_date">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res.qrc">
      <normaloff>:/menu/res/date.png</normaloff>:/menu/res/date.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="DatagraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>480</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>640</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>640</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView{
border:0px solid white;
background-color : rgb(244, 244, 244);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>20</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#line{
	background-color: rgb(244, 244, 244);;
    color:rgb(244, 244, 244);;
    border-left: 2px dotted rgb(42, 180, 255);

}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
